packages/core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -24,14 +24,22 @@ export const DatePicker: React.FC<Props> & WithStyle = React.memo(
                 popoverPlacement,
                 minSelectableDate,
                 maxSelectableDate,
+                mobile,
                 ...restProps
             } = props,
             id = props.id || props.label.toLowerCase().replace(/\s/g, '') || 'medly-datepicker', // TODO:- Remove static ID concept to avoid dup ID
             date: Date | null = useMemo(
                 () => (value instanceof Date ? value : typeof value === 'string' ? parseToDate(value, displayFormat) : null),
                 [value, displayFormat]
             );
+        const convertDate = (d: string) => {
+            const [year, month, day] = d.split('-');
+            return month + '/' + day + '/' + year;
+        };
 
+        const handleOnChangeNative = (event: React.FormEvent<HTMLInputElement>) => {
+            return parseToDate(convertDate(event.currentTarget.value), displayFormat);
+        };
```

Comment:
Might want to wrap these in `useCallback` for consistency

Reply:
https://github.com/medly/medly-components/pull/186/commits/42042fa56f33c374c281b5d1b6c37c157197f2aa